README.md

```diff
@@ -41,4 +41,3 @@ As Flowstate and the SDK are in beta, the contents of this repository are
 subject to change.
 Use of this repository requires participation in the beta for Intrinsic
 Flowstate, which is accepting [applications](https://intrinsic.ai/beta).
-Access to this repository is subject to the associated [LICENSE](LICENSE).
```

Comment:
For the reviewers, please take special note of this change. I think it's appropriate, but wanted a second opinion.

Reply:
(Maybe the text above could use rewording too?)